packages/agent/app/harness.ts

```diff
@@ -1,3 +1,14 @@
+import 'regenerator-runtime/runtime';
```

Comment:
Man, having to add this thing here is a real bummer.

Reply:
Yeah, this really sucks. I guess the upshot is that we'll have an expanded base of browser support 🤷